src/setup-dotnet.ts

```diff
@@ -90,9 +93,14 @@ export async function run() {
       }
     );
 
-    core.setOutput('dotnet-version', versionToOutput);
+    core.setOutput(Outputs.DotnetVersion, versionToOutput);
 
-    const matchersPath = path.join(__dirname, '..', '.github');
+    if (core.getBooleanInput('cache') && isCacheFeatureAvailable()) {
+      const cacheDependencyPath = core.getInput('cache-dependency-path');
+      await restoreCache(cacheDependencyPath);
+    }
+
+    const matchersPath = path.join(__dirname, '../..', '.github');
```

Comment:
Should not "../.." be os independent? 
```
path.join(__dirname, '..','..', '.github');
```
or `path.sep` ?

Reply:
Hi, @nogic1008 👋 May I draw your attention to the [comment](https://github.com/actions/setup-dotnet/pull/303#discussion_r1197755769) ? If you don't have capacity to solve it, just ping me, I'll do it by myself.

Reply:
Fixed. (also [src/installer.ts](https://github.com/actions/setup-dotnet/blob/39d3726ca6f648bd8a27ab89bb2b183a2153322e/src/installer.ts#L191))